packages/terra-notification-dialog/src/NotificationDialog.module.scss

```diff
@@ -42,98 +33,120 @@
+    flex-shrink: 1;
+    margin-left: var(--terra-notification-dialog-header-container-margin-left, 0.5rem);
+  }
 
-    > .error {
-      background: var(--terra-notification-dialog-error-icon-background, inline-svg("<svg viewBox='0 0 49 49' xmlns='http://www.w3.org/2000/svg'><path fill='#e50000' d='M24 0c13.3 0 24 10.7 24 24S37.3 48 24 48 0 37.3 0 24C0 10.8 10.7 0 23.9 0h.1z'/><path fill='#fff' d='M7 21.5h34v5H7v-5z'/></svg>"));
-    }
+  .signal-word {
+    color: var(--terra-notification-dialog-signal-word-color, #1c1f21);
+    font-size: var(--terra-notification-dialog-signal-word-font-size, 1.42857rem);
+    font-weight: bold;
+    letter-spacing: -0.018rem;
+    line-height: 1.15;
+    min-height: var(--terra-notification-dialog-signal-word-min-height, 1.64285rem);
   }
 
   .title {
-    color: var(--terra-notification-dialog-title-color, #000);
-    flex: 0 0 auto;
-    flex-wrap: wrap;
-    font-size: var(--terra-notification-dialog-title-font-size, 1.07142rem);
-    font-weight: var(--terra-notification-dialog-title-font-weight, bold);
-    margin-top: var(--terra-notification-dialog-title-margin-top, 0.21428rem);
-    text-align: var(--terra-notification-dialog-title-text-align, left);
+    color: var(--terra-notification-dialog-title-color, #262626);
+    font-size: var(--terra-notification-dialog-title-font-size, 1.2857rem);
+    font-weight: var(--terra-notification-dialog-title-font-weight, normal);
+    letter-spacing: -0.018rem;
+    line-height: 1.16667;
+    margin: var(--terra-notification-dialog-title-margin, 0);
+    min-height: var(--terra-notification-dialog-title-min-height, 1.5rem);
     width: 100%;
     word-wrap: break-word;
   }
 
+  .body {
+    background-color: var(--terra-notification-dialog-body-background-color, #fff);
+    flex: 1 1 auto;
+    height: 100%;
+    min-height: var(--terra-notification-dialog-body-min-height, 6.42857rem);
+    overflow: auto;
+    padding: var(--terra-notification-dialog-body-padding, 0.85714rem 1.64285rem 1.07142rem);
+    width: 100%;
+  }
+
   .message {
     color: var(--terra-notification-dialog-message-color, #383f42);
-    flex: 0 0 auto;
-    flex-wrap: wrap;
     font-size: var(--terra-notification-dialog-message-font-size, 1rem);
+    letter-spacing: 0.007rem;
+    line-height: 1.42857;
+    padding: var(--terra-notification-dialog-message-padding, 0.35714rem 0);
     text-align: var(--terra-notification-dialog-message-text-align, left);
     width: 100%;
     word-wrap: break-word;
   }
 
-  .title + .message {
-    padding-top: var(--terra-notification-dialog-message-padding-top, 1.07142rem);
+  .footer {
+    background-color: var(--terra-notification-dialog-footer-background-color, #f9f9f9);
+    border-top: var(--terra-notification-dialog-footer-border-top, 1px solid #dedfe0);
+    flex: 0 0 auto;
+    font-size: var(--terra-notification-dialog-footer-font-size, 0.8rem);
+    min-height: var(--terra-notification-dialog-footer-min-height, 0.71428rem);
```

Comment:
Footer is missing padding that used to be on the container: 

https://github.com/cerner/terra-framework/blob/202716959067f01c1621679a483557781a2f7580/packages/terra-notification-dialog/src/NotificationDialog.module.scss#L59

Though I wonder if we should more directly match action-footer construction (majority of space is on the footer-container with only margin-left being applied to the buttons, hmm. (ignore this statement for now)